jenkins/buster-chromebooks.jpl

```diff
@@ -0,0 +1,24 @@
+@Library('kernelci') _
+import org.kernelci.debian.RootFS
+
+/* ----------------------------------------------------------------------------
+ * Jenkins parameters
+
+DOCKER_BASE
+  Dockerhub base address used for the build images
+
+*/
+
+def r = new RootFS()
+
+def config = ['name':"buster-chromebooks",
```

Comment:
This rootfs doesn't actually have anything specific to Chromebooks.  Instead, it just has extra Python packages.  So I think we should call it `buster-python` instead to use it with any test suite that requires Python.

Reply:
The purpose is to include the chromebooks test in this rootfs, hence the name includes chromebooks. But as you know these are still under development so our plan is to use that rootfs as a base and clone the test suite meanwhile. We will end with a buster-chromebooks rootfs so if nobody is interested in python right now I am not sure makes sense to rename. Anyway, both solutions work for me now, so just tell what do you prefer.

Reply:
OK, then maybe it should be named with the test suite it will include.  If it's for cros-ec then maybe use that in the name (e.g. `buster-cros-ec`) rather than `buster-chromebook`.